xml/admin_ceph_gateway.xml

```diff
@@ -3234,7 +3239,37 @@ rgw_frontends="beast port=80"
 rgw_zone=us-east-2</screen>
 <screen>&prompt.cephuser;sudo systemctl start ceph-radosgw@rgw.us-east-2</screen>
    </sect3>
-  </sect2>
+ <sect3 xml:id="ceph-update-dashboard-config">
+   <title>Update &dashboard; Configuration</title>
+   <procedure>
+     <step>
+       <para>Set the following to update your dashboard configuration for
+       multi-site:</para>
+<screen>&prompt.cephuser;ceph dashboard set-rgw-api-access-key <replaceable>ACCESS-KEY</replaceable>
+&prompt.cephuser;ceph dashboard set-rgw-api-secret-key <replaceable>SECRET-KEY</replaceable>
+</screen>
+      <para>Based on the secret keys and access key variables.</para>
+     </step>
+     <step>
+       <para>Set the dashboard default to <literal>admin</literal>:</para>
+<screen>&prompt.cephuser;ceph dashboard set-rgw-api-user-id admin</screen>
+     </step>
+     <step>
+       <para>Disable and re-eanble the dashboard to apply settings.</para>
+<screen>&prompt.cephuser;ceph mgr module disable dashboard
+&prompt.cephuser; ceph mgr module enable dashboard
+</screen>
+     </step>
+     <step>
+       <para>If the &ogw; node changed or a load balancer is used instead of
+         the &ogw;s, update the dashboard:</para>
+<screen>&prompt.cephuser;dashboard set-rgw-api-host HOST
```

Comment:
I'm not familiar with this XML format, but i think this should be
```suggestion
<screen>&prompt.cephuser;dashboard set-rgw-api-host <replaceable>HOST</replaceable>
```

Reply:
Correct! Thanks @votdev :)